### PR TITLE
Remove dead code

### PR DESCRIPTION
**Pull request by christophermaier**
DO NOT MERGE until after https://github.com/habitat-sh/habitat/pull/4700 merges and the `Cargo.lock` dependency is bumped!

Signed-off-by: Christopher Maier <cmaier@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: test/builder-api/src/ext.js [hunk 1]
```diff
@@ -7,23 +7,6 @@ const installationId = 56940;
...
-        .end(function(err, res) {
-          expect(res.text).to.be.empty;
-          done(err);
-        });
-    });
-
-    // This one requires correct GH creds to test. Authenticating with legit GH
-    // creds would break the janky session short-circuiting we have for these
-    // tests. A bit of a catch-22.
-    it('succeeds');
```
Comment (raskchanky) [inline test/builder-api/src/ext.js]: Not gonna lie - I'm pretty stoked to see this deleted 😁

Comment (fnichol) [reply test/builder-api/src/ext.js]: I bet--a call to an external service in a test? One of the hardest things!

**Comment by christophermaier** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/builder/builds/348415039).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/builder/builds/348415039).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**